bash/code_to_gast/bash_main.js

```diff
@@ -0,0 +1,14 @@
+const bash_router = require("./bash_router.js")
+const parse = require("bash-parser");
+
+const bash_input = process.argv[2]
```

Comment:
Should this fail when given too little arguments?

Reply:
This will always be given three arguments, `['node', 'bash/code_to_gast/bash_main.js', input_code]`. The third argument is always the input_code

Reply:
What if the integration is accidentally broken and it's given incorrect args? Solid validation could prevent silent errors.